app/templates/home.html

```diff
@@ -15,9 +15,16 @@ <h3 ng-click="home.selectRoom(room)">{{ room.name }}<h3>
     <h1>{{ home.currentRoom.name }}</h1>
         <div ng-repeat="message in home.messages">
             <div class="time"> {{ message.time | date:'short'}}</div>
-            <div class="username">{{ message.username }}</div>
+            <div class="username">{{ home.user }}</div>
```

Comment:
This is a problem--it will always show the messages coming from the current user, not the user who posted the message. You can see this by opening the page in another browser--each browser will show the messages as written by the username of that browser.